src/table.css

```diff
@@ -0,0 +1,56 @@
+@layer components {
```

Comment:
Nice I'll be able to use this on the table stuff I did. Maybe we can find a way to get it into holocene?